staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -438,15 +490,28 @@ func (p *applyPatcher) applyPatchToCurrentObject(obj runtime.Object) (runtime.Ob
 		return nil, errors.NewBadRequest(fmt.Sprintf("error decoding YAML: %v", err))
 	}
 
-	return p.fieldManager.Apply(obj, patchObj, p.options.FieldManager, force)
+	obj, err := p.fieldManager.Apply(obj, patchObj, p.options.FieldManager, force)
+	if err != nil {
+		return obj, err
+	}
+
+	if p.validationDirective == metav1.FieldValidationStrict || p.validationDirective == metav1.FieldValidationWarn {
```

Comment:
non-blocking for this PR, but for beta, spawn something to track deciding whether a fieldValidation=Strict fatal error should return _before_ an error from the apply operation

Reply:
TODO added